crypto/dsa/dsa_lib.c

```diff
@@ -137,6 +137,8 @@ static DSA *dsa_new_intern(ENGINE *engine, OSSL_LIB_CTX *libctx)
     if (ret == NULL)
         return NULL;
 
+    ossl_ffc_params_init(&ret->params);
+
```

Comment:
Similarly to above comment, move it further down.

Reply:
Done, see [force-push diff](https://github.com/openssl/openssl/compare/8e0f3474abc5a53bc6304c6b3157a6b09c5cdfac..7febe199db6310c09d8f398e42c8bf95f63755ff).